input/resources/StructureDefinition-cqf-numeratorMembership.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="cqf-numeratorMembership"/>
+  <url value="http://hl7.org/fhir/StructureDefinition/cqf-numeratorMembership"/>
+  <version value="1.0.0"/>
+  <name value="DEQMCalculatedNumeratorMembership"/>
+  <title value="DEQM Calculated Numerator Membership"/>
+  <status value="active"/>
+  <experimental value="false"/>
+  <date value="2024-07-29T00:00:00-07:00"/>
+  <publisher value="Da Vinci Project"/>
+  <contact>
+    <telecom>
+      <system value="url"/>
+      <value value="http://www.hl7.org/Special/committees/cqi/index.cfm"/>
+    </telecom>
+  </contact>
+  <description value="The numerator population membership is the number of subjects in the numerator after exceptions and exclusions. See ([https://hl7.org/fhir/us/cqfmeasures/measure-conformance.html](https://hl7.org/fhir/us/cqfmeasures/measure-conformance.html)) for further discussion."/>
+  <jurisdiction>
+    <coding>
+      <system value="urn:iso:std:iso:3166"/>
+      <code value="US"/>
+      <display value="United States of America"/>
+    </coding>
+  </jurisdiction>
+  <fhirVersion value="4.0.1"/>
+  <kind value="complex-type"/>
+  <abstract value="false"/>
+  <context>
+    <type value="element"/>
+    <expression value="MeasureReport.group"/>
+  </context>
+  <type value="Extension"/>
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension"/>
+  <derivation value="constraint"/>
+  <differential>
+    <element id="Extension">
+      <path value="Extension"/>
+      <short value="The number of subjects in the numerator after exceptions and exclusions"/>
+      <definition value="The numerator population membership is the number of subjects in the numerator after exceptions and exclusions."/>
+      <min value="0"/>
+      <max value="1"/>
+    </element>
+    <element id="Extension.url">
+      <path value="Extension.url"/>
+      <type>
+        <code value="uri"/>
+      </type>
+      <fixedUri value="http://hl7.org/fhir/StructureDefinition/cqf-numeratorMembership"/>
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]"/>
+      <short value="The number of subjects in the numerator after exceptions and exclusions."/>
+      <min value="0"/>
+      <max value="1"/>
+      <type>
+        <code value="integer"/>
```

Comment:
The https://jira.hl7.org/browse/FHIR-44736 resolution says "0..1 Quantity". Change integer to Quantity.

Reply:
see above

Reply:
Changed to Quantity for now